UPGRADE.md

```diff
@@ -8,14 +8,7 @@
 
 ```bash
 brew upgrade vips
-brew upgrade glide
 brew upgrade golang 
-glide update
```

Comment:
@rgritti what's the go modules version of this? Is there a way to automatically update all dependencies?

Reply:
there isn't.
When you run `go test ./..` of `go build ./...` it will first update the dependencies